Frontend/src/App.css

```diff
@@ -15,18 +15,24 @@
   --lime: #E5E216;
   --sage: #C0C382;
   --orange-red: #D93F24;
-  --default-background: white;
-  --error-color: red;
+
 }
+  --error-color: red;
```

Comment:
The `--error-color: red;` should be in the `:root` block above it. If you notice an issue with the height of the sidebar, it's because of this.